README.md

```diff
@@ -16,9 +16,10 @@ This GitHub Action instruments your workflows using [Honeycomb's buildevents too
 
 ### 📣 Adopting version 2.0.0
 
-- The required input field `job-status` has been renamed to `status`. This was done because status now can be job's or the workflow's. 
+- The input field `job-status` has been renamed to `status`. This no longer required in every job. This was done because status now can be job's or the workflow's. 
```

Comment:
```suggestion
- The input field `job-status` has been renamed to `status`. This is no longer required in every job. This was done because status can now be job status or workflow status. 
```